### DISCUSSION TITLE
exporting to star file

### DISCUSSION THREAD

**Discussion by DrJesseHansen** [nextpyp/.github#81] [Q&A] [answered]
hi,

Is it possible to do the full preprocessing block then export the particle coordinates as a star file, or equivalent?  Thank you!

Jesse

**Comment by JuhaHuiskonen** [comment, marked as answer]
This is what I did: 
1. Run a dummy refinement (skip-refinement and no constrained refinement done either)
2. Create a second refinement block. Enable the export option in the second block and read the par file (in the frealing folder) from the first block.
